### PR TITLE
Feature/form styles and functionality

### PR DESCRIPTION
**Pull request by narendra28051991**
## Ticket Link: [ID](https://www.meistertask.com/app/task/ID)
[ID: 8P8eVMhE](https://www.meistertask.com/app/task/8P8eVMhE)
[ID: vNTyAmbV](https://www.meistertask.com/app/task/vNTyAmbV)
[ID: fzwjqi1L](https://www.meistertask.com/app/task/fzwjqi1L)

## 📝 Description
Created one style sheet for all the forms
Firebase storage will allow multiple lesson images

- [x] Description of work
The three forms will be responsive for all screen resolutions. Also the lessons url's will be replaced by lesson pictures.
The firebase storage and axios are commented as they will be tested by Nelia.
The user and the courses are made reusable.

### CODE REVIEW

**Review by Nelia11 — COMMENTED** [review]
Looks good to me!

File: frontend/src/types/component.d.ts
Comment (jawkha) [inline frontend/src/types/component.d.ts:1]: Any reason for using `snake case` for naming some properties and `camel case` for others?

File: frontend/src/pages/myCourses/MyCourses.tsx
Comment (jawkha) [inline frontend/src/pages/myCourses/MyCourses.tsx:1]: Are the courses still not fetched from the database?

**Review by jawkha — CHANGES_REQUESTED** [review]

File: frontend/src/types/component.d.ts
Comment (narendra28051991) [reply frontend/src/types/component.d.ts:1]: Yes Jawad because of the backend

**Review by jawkha — APPROVED** [review]

File: frontend/src/pages/myCourses/MyCourses.tsx
Comment (jawkha) [reply frontend/src/pages/myCourses/MyCourses.tsx:1]: @narendra28051991 Please create a separate issue for this to ensure that courses are fetched from the database if this is not already covered in another PR.

**Review by jawkha — APPROVED** [review]

**Pull request merged**